### PR TITLE
Viacheslav blazhkov lesson 35

### PR DESCRIPTION
**Pull request by ViacheslavBlazhkov**

### CODE REVIEW

File: ViacheslavBlazhkov/Lesson35/Middlewares/ThrowException.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿using System.Text;
+
+namespace Lesson35.Middlewares
+{
+    public class ThrowException
```
Comment (denys-kniaziev) [inline ViacheslavBlazhkov/Lesson35/Middlewares/ThrowException.cs]: What is purpose of this middleware? It doesn`t throw exception or handle it

**Review by denys-kniaziev — CHANGES_REQUESTED** [review]
One of the filters should be attribute based

File: ViacheslavBlazhkov/Lesson35/Startup.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            });
+        }
+
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+            //app.UseMiddleware<ThrowException>();
```
Comment (denys-kniaziev) [inline ViacheslavBlazhkov/Lesson35/Startup.cs]: Why comment?

File: ViacheslavBlazhkov/Lesson35/Middlewares/ThrowException.cs (hunk 1 above)
Comment (ViacheslavBlazhkov) [reply ViacheslavBlazhkov/Lesson35/Middlewares/ThrowException.cs]: This middleware threw exception at start of program. Now I change it and middleware redirect us to other page if address contain /swagger and page where we want do go does not exist

**Review by denys-kniaziev — APPROVED** [review]

**Pull request merged**